packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -342,7 +346,7 @@ export default class TableResize implements EditorPlugin {
     };
 
     private resizeTable = (e: MouseEvent) => {
-        if (this.currentTd) {
+        if (this.currentTd && this.resizingState !== ResizeState.None) {
```

Comment:
In what case resizingState is not None?

Reply:
This is to fix the issue where it would do vertical resizing after user resized the cell horizontally. Since in endResizeTable, we are setting resizingState as None, but in resizeTable it was only checking if Horizontal or not (any state that is not horizonal is considered as vertical), which is the root cause of this issue.